### PR TITLE
TraceSVG color change Proposal

### LINKED ISSUE #1018: Update traceSVG colors
Right now, we trace all of the image fluids with the same color: #A7DEF6. Let's be a little more intentional about which color we choose for each image so the trace color roughly corresponds to the image color.

- [ ] `src/components/pages/homepage/datasets.js:12`
- [ ] `src/components/pages/homepage/datasets.js:20`
- [ ] `src/components/pages/homepage/large-dataset.js:13`
- [ ] `src/components/pages/homepage/large-project.js:15`
- [ ] `src/components/pages/race/header-hero.js:15`
- [ ] `src/pages/about-data/visualization-guide/index.js:299`

### PR DESCRIPTION
**Pull request by Melvin-Viana**
Fixes #1018 
- Didn't use that threshold option that @kevee described.
- I only did the first 5 of the images and used colors used within the images.

Couldn't find ` src/pages/about-data/visualization-guide/index.js:299`

It was hard to view changes in my local environment so sorry if it doesn't look too good. Let me know what you guys think.

### CODE REVIEW

**Comment by kevee** [comment]
Hi, @Melvin-Viana! If you use dev tools to slow your connection speed you'll be able to preview the SVG tracing easier.

**Comment by schwartzadev** [comment]
Thanks for taking a stab at this! The visualization guide image was on another branch not yet merged into master -- you weren't missing anything.

File: src/components/pages/homepage/datasets.js [hunk 1]
```diff
@@ -9,15 +9,15 @@ export default () => {
...
+          fluid(maxWidth: 1200, traceSVG: { color: "#182E4E" }) {
             ...GatsbyImageSharpFluid_tracedSVG
           }
         }
       }
       covidExit: file(relativePath: { regex: "/covid-exit-strategy.png/" }) {
         relativePath
         childImageSharp {
-          fluid(maxWidth: 1200, traceSVG: { color: "#A7DEF6" }) {
+          fluid(maxWidth: 1200, traceSVG: { color: "#F7CB82" }) {
```
Comment (schwartzadev) [inline src/components/pages/homepage/datasets.js:20]: Can we make this a dark color, like maybe sample the black from the `State's progress...` headline?

Right now, this renders in orange, which is a little distracting.

File: src/components/pages/homepage/datasets.js [hunk 2]
```diff
@@ -9,15 +9,15 @@ export default () => {
...
+          fluid(maxWidth: 1200, traceSVG: { color: "#182E4E" }) {
             ...GatsbyImageSharpFluid_tracedSVG
           }
         }
       }
       covidExit: file(relativePath: { regex: "/covid-exit-strategy.png/" }) {
         relativePath
         childImageSharp {
-          fluid(maxWidth: 1200, traceSVG: { color: "#A7DEF6" }) {
+          fluid(maxWidth: 1200, traceSVG: { color: "#020202" }) {
```
Comment (Melvin-Viana) [inline src/components/pages/homepage/datasets.js:20]: Added the change to use the shade of black used in the graph heading.

**Comment by Melvin-Viana** [comment]
@kevee thanks I never knew you could do that with dev tools.

Thank you both for being so friendly and allowing an opportunity to help out this project

**Pull request merged**